### DISCUSSION TITLE
SolutionUserObject and dynamics

### DISCUSSION THREAD

**Discussion by TheGreatCid** [idaholab/moose#26275] [Q&A General]
Hi all,

Does MOOSE currently have the capability to use SolutionUserObject to start a dynamic simulation where u_old is not zero. 
I am doing this in the context of remeshing and restarting, so I cannot use the restart/checkpoint system.

Essentially, If I output the u_old values in the file I am reading with SolutionUserObject, can I populate the u_old values as in initial condition in the new simulation?
Is there a better way to go about this?

**Comment by lindsayad** [comment]
Yea you could write out `foo_old` and read in `foo_old`
